react/containers/withGoogleMaps.js

```diff
@@ -13,6 +15,18 @@ export function withGoogleMaps(ComponentToWrap) {
 
     render() {
       const { googleMapsKey, intl } = this.props
+
+      if (!googleMapsKey) {
+        return (
+          <ComponentToWrap
+            {...this.props}
+            googleMaps={null}
+            loading={false}
```

Comment:
could you just not pass the `loading` props in order to get it false in `ComponentToWrap`?